family2family/src/main/java/org/cbtf/f2f/model/Subcategory.java

```diff
@@ -34,8 +34,11 @@
     @JoinColumn(name = Category.COLUMN_CATEGORY_ID)
     private Category category;
 
-    @OneToMany(mappedBy = ChildSubcategory.FIELD_SUBCATEGORY)
-    private List<ChildSubcategory> childSubcategories;
+    public Subcategory(String description, Boolean active, Category category) {
```

Comment:
http://localhost:8080/internal-api/Subcategory errors. Complains about not having a default constructor.